Fix for broken GSI documentation (#248)
couchbasecloud/terraform-provider-couchbase-capella: docs/resources/cluster.md

Required:

- `compute` (Attributes) (see [below for nested schema](#nestedatt--service_groups--node--compute))

> - `disk` (Attributes) The 'storage' and 'IOPS' fields are required for AWS. For Azure, only the 'disktype' field is required, and for Ultra disk type, you can provide all 3 - storage, iops and autoexpansion fields. For Premium type, you can only provide the autoexpansion field, others can't be set.In the case of GCP, only 'pd ssd' disk type is available, and you cannot set the 'IOPS' field. (see [below for nested schema](#nestedatt--service_groups--node--disk))

Reviewer: ```suggestion
- `disk` (Attributes) For AWS, the 'storage' and 'IOPS' fields are required. For Azure, only the 'disktype' field is required. For Ultra disk type, you can provide all 3 fields: 'storage', 'IOPS' and 'auto-expansion'. For Premium type, you can set the 'auto-expansion' field only. For GCP, only the 'pd ssd' disk type is available, and you cannot set the 'IOPS' field. (See [below for nested schema](#nestedatt--service_groups--node--disk))
```
